src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,836 @@
+{
```

Comment:
<blockquote>In this part, we will solve two simple MDO problems<code>,</code> which consists in finding the aspect ratio which respectively gives the lowest MTOW for the given mission and the lowest fuel consumed.</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>